### PR TITLE
gh-105373: Elaborate Pending Removal in What's New in Python 3.13

### PR DESCRIPTION
**Pull request by vstinner**
* Issue: gh-105373

----
:books: Documentation preview :books:: https://cpython-previews--106675.org.readthedocs.build/

### CODE REVIEW

**Comment by vstinner** [comment]
cc @CAM-Gerlach @hugovk

File: Doc/whatsnew/3.13.rst [hunk 1]
```diff
@@ -395,9 +397,14 @@ although there is currently no date scheduled for their removal.
 * :mod:`os`: Calling :func:`os.register_at_fork` in multi-threaded process.
 
 * :class:`!pydoc.ErrorDuringImport`: A tuple value for *exc_info* parameter is
-  deprecated, use an exception instance.
-
-* :mod:`re`: bad character in group name.
+  deprecated, use an exception instance
```
Comment (hugovk) [inline Doc/whatsnew/3.13.rst]:
```suggestion
  deprecated, use an exception instance.
```

**Review by hugovk — APPROVED** [review]

**Comment by vstinner** [comment]
Merged, thanks for the review.

**Comment by hugovk** [comment]
Thanks for updating it!

**Pull request merged**